device_doctor/tool/build.bat

```diff
@@ -0,0 +1,28 @@
+:: Copyright 2020 The Flutter Authors. All rights reserved.
+:: Use of this source code is governed by a BSD-style license that can be
+:: found in the LICENSE file.
+
+REM Checks if cipd command is available.
+FOR /F "tokens=*" %%g IN ('where cipd') do (SET CIPD=%%g)
+IF %ERRORLEVEL% NEQ 0 (
+        ECHO "Please install CIPD (available from depot_tools) and add to path first.";
+        EXIT
+)
+
+REM `path` is \path\to\device_doctor\tool\
+REM `DIR` is \path\to\device_doctor\tool
+SET path=%~dp0
+SET DIR=%path:~0,-1%
+%CIPD% ensure --ensure-file %path%\ensure_file_windows -root %DIR%
```

Comment:
ensure file should be generic for the different platforms why we need something specific for windows?

Reply:
Linux and mac are using the `stable` version, but `dart2native` is not available in `stable` yet for windows.